File: scratch/inspect_meetings.mjs
const supabaseUrl = 'https://xkpqkbberckxblkhseim.supabase.co';
const supabaseKey = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InhrcHFrYmJlcmNreGJsa2hzZWltIiwicm9sZSI6ImFub24iLCJpYXQiOjE3Nzk2ODQ4NzMsImV4cCI6MjA5NTI2MDg3M30.y6cgL70Aw0bPSXlU2NgdMl3k0yBOxwHFBg9L57E79AE';

async function inspectMeetings() {
    const resM = await fetch(`${supabaseUrl}/rest/v1/meetings?select=*`, {
        headers: {
            'apikey': supabaseKey,
            'Authorization': `Bearer ${supabaseKey}`
        }
    });
    const meetings = await resM.json();
    console.log("Meetings count:", meetings.length);
    meetings.forEach(m => {
        console.log(`ID: ${m.id} | Title: ${m.title} | Type: ${m.type} | Status: ${m.status} | Minutes URL: ${m.minutes_url}`);
    });

    const resR = await fetch(`${supabaseUrl}/rest/v1/resolutions?select=*`, {
        headers: {
            'apikey': supabaseKey,
            'Authorization': `Bearer ${supabaseKey}`
        }
    });
    const resolutions = await resR.json();
    console.log("\nResolutions count:", resolutions.length);
    resolutions.forEach(r => {
        console.log(`ID: ${r.id} | Meeting ID: ${r.meeting_id} | Title: ${r.title} | Number: ${r.resolution_number}`);
    });
}

inspectMeetings();

File: scratch/inspect_meetings.py
import os
from supabase import create_client, Client
from dotenv import load_dotenv

load_dotenv()

supabase_url = os.environ.get("SUPABASE_URL")
supabase_key = os.environ.get("SUPABASE_ANON_KEY")

if not supabase_url or not supabase_key:
    print("Supabase credentials not found in env.")
    exit(1)

supabase: Client = create_client(supabase_url, supabase_key)

res = supabase.table("meetings").select("*").execute()
print("Meetings count:", len(res.data))
for m in res.data:
    print(f"ID: {m['id']} | Title: {m['title']} | Type: {m['type']} | Status: {m['status']} | Minutes URL: {m['minutes_url']}")

res_res = supabase.table("resolutions").select("*").execute()
print("\nResolutions count:", len(res_res.data))
for r in res_res.data:
    print(f"ID: {r['id']} | Meeting ID: {r['meeting_id']} | Title: {r['title']} | Number: {r['resolution_number']}")
